backend/recruitmentapp/apps/api/v1/views.py

```diff
@@ -117,3 +118,40 @@ def create(self, request):
             data=serializer.data,
             status=status.HTTP_201_CREATED
         )
+
+
+class CompetenceViewSet(viewsets.ViewSet):
+    """Viewset for applicants and recruiters to view competences.
+
+    ### Permissions
+    All authenticated users.
+    """
+
+    queryset = Competence.objects.all()
+    serializer_class = CompetenceSerializer
+    permission_classes = (IsAuthenticated, )
+
+    def list(self, request):
+        """List all competences.
+        """
+
+        queryset = Competence.objects.all()
+        serializer = CompetenceSerializer(queryset, many=True)
+        return Response(serializer.data)
+
+    def retrieve(self, request, pk=None):
+        """Retrieve a single competence.
+        """
+
+        queryset = Competence.objects.all()
+        competence = get_object_or_404(queryset, pk=pk)
+        serializer = CompetenceSerializer(competence)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    def create(self, request):
```

Comment:
This method should be either removed or restricted to only recruiters and staff. The latter requires `get_permissions()` to be overridden, as in `ApplicantViewSet`, and a new permissions class to be written.